test/manual/ldap.test.js

```diff
@@ -1,10 +1,11 @@
 'use strict';
 const { MongoClient } = require('../../src');
 const { expect } = require('chai');
+const { MongoURIError } = require('../../src/error');
 
 describe('LDAP', function () {
   if (process.env.MONGODB_URI == null) {
-    throw new Error(`skipping SSL tests, MONGODB_URI environment variable is not defined`);
+    throw new MongoURIError(`skipping SSL tests, MONGODB_URI environment variable is not defined`);
```

Comment:
no need for custom errors in tests unless the tests are intended to mock a specific sort of error

Reply:
Still needs to be updated to use the regular Error class